svthalia/concrexit
# Add note about COVID-19

### Summary
Add note about COVID-19

### How to test
Steps to test the changes you made:
1. Try to cancel for an event after the cancellation deadline.
2. See the note.
